waiter/integration/waiter/authentication_test.clj

```diff
@@ -93,46 +98,49 @@
 (deftest ^:parallel ^:integration-fast ^:explicit test-saml-authentication
```

Comment:
Don't we need to make this test unexplicit?

Reply:
needs changes that are not ready yet